### DISCUSSION TITLE
[Bug] TTS dataset has no attribute ""

### DISCUSSION THREAD

**Discussion by ytl0623** [coqui-ai/TTS#2777] [General Q&A]
### Describe the bug

Hi, I was following the [TTS_TESTING_AND_TRAINING.ipynb](https://colab.research.google.com/drive/1F8mdG6Vm7kAigyLZhc37_vTQBnh7wgk2?usp=sharing).
And when I ran to the section: _Next we will load data samples. Each sample is a list of [text, audio_file_path, speaker_name]. You can define your custom sample loader returning the list of samples._
I got the error.

### To Reproduce

1. Run [TTS_TESTING_AND_TRAINING.ipynb](https://colab.research.google.com/drive/1F8mdG6Vm7kAigyLZhc37_vTQBnh7wgk2?usp=sharing)
2. In the section:
`from TTS.tts.datasets import load_tts_samples
train_samples, eval_samples = load_tts_samples(
    dataset_config,
    eval_split=True,
    eval_split_max_size=config.eval_split_max_size,
    eval_split_size=config.eval_split_size,
)`
3. error: TTS dataset has no attribute ''

### Expected behavior

It's expected to show the number of files.
e.g., > Found 13100 files in /content/tts_train_dir/LJSpeech-1.1

### Environment

```shell
-Pytorch 1.13
-OS Linux
```

**Comment by omkarade** [comment]
Verify that the TTS dataset is correctly set up and available in the specified directory /content/tts_train_dir/LJSpeech-1.1.

Ensure that the dataset_config and other relevant parameters are properly configured and match the directory structure and filenames.

Double-check for any typos or missing attributes in the code, as the error message indicates an attribute is missing.

Check the versions of the TTS library and related dependencies to ensure compatibility.

Refer to the documentation or examples provided by the TTS library to ensure the correct usage of load_tts_samples().
